packages/nextra-theme-docs/package.json

```
@@ -31,6 +31,7 @@
   "types": "./dist/index.d.ts",
   "exports": {
     "./style.css": "./style.css",
+    "./style_node_modules.css": "./style_node_modules.css",
```

```suggestion
    "./style_node_modules.css": "./style.css",
```
And you can remove all new `*:workspace` scripts

---

It has to have two files.  Otherwise it will still be resovled as `../../packages/nextra-theme-docs/style.css`